packages/js/product-editor/src/components/feedback-bar/style.scss

```diff
@@ -117,3 +117,7 @@ $gutenberg-blue: var(--wp-admin-theme-color);
 		}
 	}
 }
+
+.woocommerce-transient-notices:has(+ .woocommerce-product-mvp-ces-footer) {
+	bottom: 68px;
+}
```

Comment:
`has` is not supported in Firefox.

This implementation, with setting the footer to `absolute`, might have issues, since it is in the admin footer slot, but would no longer be laid out where/how it would normally (that's why the notice overlaps it now).

Is there a way to do the animation without changing how it is positioned? If not, I would suggest removing the animation, or maybe changing it to a different type of animation, such as a fade-in, which wouldn't require repositioning.

Reply:
I also don't think it is ideal for this component (the feedback bar) to have any knowledge of the transient notices. The components are not connected in any way.

The original implementation avoided any connection between them.

Reply:
It's totally fine and I'm in the same page as you. But this is what it was required from the design perspective. I would like to know what @jarekmorawski has to say here so all of us will be in sync.

Reply:
Perhaps you could animate the entire `WooFooterItem.Slot`, but I'm concerned that might have issues with other potential uses of `WooFooterItem`, if the footer is intended to appear instantly in those cases.

Reply:
Yep that was the reason why I didn't do it that way. We don't want other Fills within that Slot to be animated too.

Reply:
Can you record a video and show me what the problem is? Ideally, the snackbar notifications would slide out from the bottom of the screen and appear above the feedback bar.

Reply:
@jarekmorawski the problem is not visual, it is more a technical concern. The notice and the feedback bar have the side out from the bottom animation and in order to colocate each in a correct way they need to know about each other, which from @mattsherman and my perspectives is not correct.

Reply:
Interesting. Based on my testing, the feedback bar shows up after the product is saved. AFAIK, it has no animation, meaning the snackbar could slide out from behind it.

https://github.com/woocommerce/woocommerce/assets/79307566/485d6833-9f73-43e0-84bd-123d163c7a32

Reply:
> Interesting. Based on my testing, the feedback bar shows up after the product is saved.

It has a little delay so it can be perceived like that. In WC 8.0 it does has animation and it overlaps with the transient notices. This PR is expected to solve the overlapping problem described in #39228, but it introduces firefox compatibility issues and the responsibility problem mentioned before.

Reply:
What if we removed the animation from the feedback bar? Would it help? If so, I'm up for it.